backend/qemu.pm

```diff
@@ -874,6 +874,13 @@ sub start_qemu {
         for (my $i = 0; $i < $num_networks; $i++) {
             if ($vars->{NICTYPE} eq "user") {
                 my $nictype_user_options = $vars->{NICTYPE_USER_OPTIONS} ? ',' . $vars->{NICTYPE_USER_OPTIONS} : '';
+                if ($vars->{QEMU_ENABLE_SMBD}) {
+                    my $smbd_bin = find_bin('/usr/sbin/', 'smbd');
+                    $smbd_bin                             or die "No server to provide SMB/CIFS services to SUT";
+                    (qx($smbd_bin --version) =~ /(\d.*)/) or die "smbd version was not matched";
+                    bmwqemu::diag "Worker is using smbd of version: $1";
```

Comment:
The use of "Worker" is a little bit confusing here as we usually mean the openQA worker in this context. I'd rather log "smbd version found on worker host: ".